src/pages/dashboard/components/EpisodeDetails.tsx

```diff
@@ -66,10 +66,38 @@ function EpisodeDetails({ episode, isInCollection = false, showDate = false }: P
     return `${((resumePosition.asMilliseconds() / duration.asMilliseconds()) * 100).toFixed(2)}%`;
   }, [episode.Duration, episode.ResumePosition]);
 
+  const anidbEpisodeAlias = (type: EpisodeTypeEnum, epTitle: string) => {
```

Comment:
This function seems like it can be moved outside the `EpisodeDetails` component, as it doesn't directly use any of the variables from it's parent scopes.